### ISSUE TITLE
Comment emails use HTTP web addresses

### ISSUE THREAD

**Issue by malemburg** [python-jobs-board/pythondotorg#24]
Example:

```
Marc (comment),

There is a new comment following up yours.

Post: 
URL:  http://python.org/newjobs/146/
Sent by: Marc-Andre Lemburg, 06/04/2014

--- Comment: ---
The Python Job Board is for paid positions only. The Python Community has a separate Wiki for listing voluntary positions available at http://wiki.python.org/moin/VolunteerOpportunities.

Comment by mal@egenix.com


-----
Click on the following link to mute the comments thread. You will no longer receive follow-up notifications:

http://python.org/comments/mute/.eJxdUttu0zAYTtet3bwTY-N8WDiMpSBFK2JI3CH1MhpIZUi-sxzbaz2cOL_tDIo0CW6geyuehGfBcSOQkCzZ3_d__9nfuozpambEBBEjmC6tMzVz2iCIGD-n5UQTpotClM6SL46nhey1oBB2W11I5WVqk83PBPLX07id0x7B8CSsJXouiyE2NoJxIDr2sF5iKGu_dMH3PIM9I60so9IXgsHqFtzzTZiaVM41wzQtXPK3EhVCAsgivenSmldKf6wrW5yGwD1PVuZIMNuZBrg0XBjazDl5qzBy22hJqKwwpaSFgO6Q7oYbFSZt0ANdCVUEkCioV7ARVQdXbauamuk3P9XKezixKPTqbSxPzR2wrq4Ncb5LP7PeQ9vB-fSNW26WSWaPm9kGyFoSO3xzex96DTg2ii4hT2K4DZe95et80I6wqkTcIc1l5OFQH8fcHeE-r97y3vHSTT_BfeYz_8VkY-nIwT3B2N48AMejmE_TGSxP1J9ghh3PR6-eg2PFsupCOXcCGvhMd4MpqP05XCYDo_SN8fwBPc8R6xrqnja_pyU5-1_SXNqBTpp3h8aCYKDwXd4dgmHSXD0oqrIjnMAgD4OKM1srB8zoP2EoXBvQi69R5-gczU9AU.z4gIBizaC8ZAQNOPDGvOECJdGHA

--
Kind regards,
python.org
```

**Comment by malemburg** [comment]
They should use https://www.python.org/ as web address.

Note: This may be an issues in the deployment, which we cannot fix in the site code.

**Comment by SarahPythonista** [comment]
Fixed upstream with python/pythondotorg@10c5e6761b60c2dc246a3e23d12ce836b350b4bb
